pkg/config/config_test.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadConfig(t *testing.T) {
```

Comment:
For the changes I made, I thought it was out of scope in terms of testing to validate the contents of the YAML file, i.e. this but at the end of LoadConfig.

```go
err = yaml.Unmarshal(data, config)
if err != nil {
	return nil, fmt.Errorf("error unmarshaling config: %w", err)
}
```